.travis.yml

```diff
@@ -7,16 +7,14 @@ env:
 matrix:
   include:
   - os: linux
-    dist: trusty # Ubuntu 14.04
-    dotnet: 2.1.300
+    dist: focal # Ubuntu 20.04
+    dotnet: 5.0.408
```

Comment:
note: i'm moving this to .NET 6 in 1.9.1, required for M1 compatibility.

Reply:
Yeah, good!